tests/FileStrategy.test.ts

```diff
@@ -0,0 +1,196 @@
+import { MemoryStrategy } from '../src/MemoryStrategy';
+import { FileStrategy } from '../src/FileStrategy';
+import * as fileDependency from '../src/utils/file';
+
+describe('getAll', () => {
+  test('Call readDocuments function', async () => {
+    const pathToFile = '/path/to/file';
+
+    const mockReadDocuments = jest.spyOn(fileDependency, 'readDocuments');
```

Comment:
spyOn is totally ok. if you have difficulties with solving any babel errors you can try to use either mock or doMock, so fs, fs-extra dependencies are not imported